src/reducers/books.js

```diff
@@ -0,0 +1,38 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+
+const initialState = [
+  {
+    title: 'The power of habit',
+    category: 'self improvement',
+    id: Math.floor(Math.random() * 1000),
+  },
+  {
+    title: 'The prince',
+    category: 'Politics',
+    id: Math.floor(Math.random() * 1000),
+  },
+  {
+    title: 'The everything store',
+    category: 'Biography',
+    id: Math.floor(Math.random() * 1000),
+  },
+  {
+    title: "The Hitchhiker's Guide to the Galaxy",
+    category: 'Sci-Fi',
+    id: Math.floor(Math.random() * 1000),
+  },
+];
```

Comment:
According to the pathwright, you are supposed to create the book object inside `src/index.js`.
screenshot below;
![image](https://user-images.githubusercontent.com/31897434/83743295-70e7f400-a652-11ea-8474-506494691d91.png)